pkg/apis/serving/v1alpha1/revision_defaults_test.go

```diff
@@ -130,6 +130,41 @@ func TestRevisionDefaulting(t *testing.T) {
 				},
 			},
 		},
+	}, {
+		name: "lemonade",
+		wc:   v1beta1.WithUpgradeViaDefaulting,
+		in: &Revision{
+			Spec: RevisionSpec{
+				DeprecatedContainer: &corev1.Container{
```

Comment:
Can you add a test that sets Resources and Timeout seconds in the Deprecated container and makes sure it gets copied over correctly?